### DISCUSSION TITLE
Possible to use it with Codeigniter 4

### DISCUSSION THREAD

**Discussion by rupomkhondaker** [php-flasher/php-flasher#205] [Ideas]
Possible to use with CodeIgniter 4 ?

**Comment by yoeunes** [comment]
Hi Rupom,

I haven’t personally worked with CodeIgniter before. However, you can still integrate PHPFlasher into your CodeIgniter project by leveraging the core components and existing framework adapters.

Here are some resources that might help you:

**Core PHPFlasher Repository:**
This is the pure PHP version of PHPFlasher, containing all the abstractions that are independent of any specific framework.
    👉 [php-flasher/flasher](https://github.com/php-flasher/flasher)
	
**Laravel Adapter:**
You can look at how the Laravel adapter is implemented to understand how to integrate PHPFlasher with a framework.
    👉 [php-flasher/flasher-laravel](https://github.com/php-flasher/flasher-laravel)
	
**Symfony Adapter:**
Similarly, the Symfony adapter can provide insights into integrating PHPFlasher with another framework.
    👉 [php-flasher/flasher-symfony](https://github.com/php-flasher/flasher-symfony)

you can get inspired to create your own adapter for CodeIgniter. If you develop a CodeIgniter adapter, I’d be more than happy to assist you or even include it in the PHPFlasher ecosystem!

**Reply by rupomkhondaker** [reply]
ok thanks I will try

**Reply by tasssman** [reply]
Do you manage integrate with CI4?

**Reply by yoeunes** [reply]
Hey @tasssman as far as I know, no one has published a CI4 adapter yet. @rupomkhondaker mentioned they would try back in November, but I haven't heard back since.

If anyone is working on this, feel free to open a draft PR if you give it a shot
